sdk/data.go

```diff
@@ -78,6 +94,17 @@ func (integration *Integration) NewMetricSet(eventType string) *metric.MetricSet
 	return &ms
 }
 
+// AddEvent method adds a new Event.
+func (integration *Integration) AddEvent(summary, category string) error {
```

Comment:
Let's assume that user doesn't want to set `category`.
With current code he will need to do it like this:
```
integration.AddEvent("MySummary", "")
```
Maybe this would be better:
```
integration.AddEvent("MySummary")
```

Reply:
@smoya Is there any easy way to this?

Proposal:
- changing function `AddEvent` which would accept as an argument `Event` object and validating inside which elements of `Event` are present and which not.
Example:
```
integration.AddEvent(sdk.Event{Summary:"summary", Category: "category"})
```
And then we don't have a problem when user doesn't want to include `category`. It would work like
```
integration.AddEvent(sdk.Event{Summary:"summary"})
```
Also in the future when we add more fields to the `Event` it will be easy to change `AddEvent` function.

Initially I was thinking about using variadic arguments, but then it will be hard to control it.
What do you think?

/cc @areina

Reply:
I would go for `integration.AddEvent("MySummary", sdk.DefaultEventCategory)`
where `DefaultEventCategory` is a const with the value `notifications`.

Also I would add the suggested categories listed on the docs: https://docs.newrelic.com/docs/infrastructure/integrations-sdk/file-specifications/integration-executable-file-specifications#event-data

Reply:
I think that it won't be useful for the user having this categories list. It would be just a const right?